xcodeproj/internal/xcschemes/xcschemes.bzl

```diff
@@ -1179,6 +1179,21 @@ Address Sanitizer cannot be used together with Thread Sanitizer.
         ),
     )
 
+def _autogeneration_config(scheme_name_exclude_patterns = None):
+    """Dictionary containing scheme auto-generation configurations
+
+    Args:
+        scheme_name_exclude_patterns: Dictionary containing configuration options.
+
+    Returns:
+        A dictionary where keys are strings and the values are lists of strings
+    """
```

Comment:
Can you improve these docs, since they are user facing? In particular, `scheme_name_exclude_patterns` is a `list`, you should mention that it's regex, and ideally you provide an example as well.

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3027/commits/3c0bce7a2c8826fb7a3d511dfdbfa744e3efd5a2